anaconda/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM ubuntu:14.04
+
+MAINTAINER Travis Swicegood
+
+RUN apt-get update
+RUN apt-get install -y wget bzip2
+RUN echo 'export PATH=/opt/anaconda/bin:$PATH' > /etc/profile.d/conda.sh
+RUN wget --quiet http://repo.continuum.io/archive/Anaconda-2.1.0-Linux-x86_64.sh -O /Anaconda-2.1.0-Linux-x86_64.sh
+RUN /bin/bash /Anaconda-2.1.0-Linux-x86_64.sh -b -p /opt/anaconda
+RUN rm /Anaconda-2.1.0-Linux-x86_64.sh
+
```

Comment:
The `anaconda` installs should actually expose data volumes so you can mount this as an external source.  I'm not sure if you can expose nested volumes, but if you can I would expose:

* `/opt/anaconda/`
* `/opt/anaconda/pkgs/`
* `/opt/anaconda/envs/`

If you can't expose nested data volumes, then the top level is fine.  This is required to allow other containers to mount anaconda so they don't have to have the entire system in their image.